ote_sdk/ote_sdk/entities/model.py

```diff
@@ -110,11 +124,11 @@ def __init__(
         target_device: TargetDevice = TargetDevice.CPU,
         target_device_type: Optional[str] = None,
         optimization_type: ModelOptimizationType = ModelOptimizationType.NONE,
-        optimization_methods: List[OptimizationMethod] = None,
-        optimization_objectives: Dict[str, str] = None,
-        performance_improvement: Dict[str, float] = None,
+        optimization_methods: Optional[List[OptimizationMethod]] = None,
+        optimization_objectives: Optional[Dict[str, str]] = None,
+        performance_improvement: Optional[Dict[str, float]] = None,
         model_size_reduction: float = 0.0,
-        _id: Optional[ID] = None,
+        _id: Optional[Union[ID, ObjectId]] = None,
```

Comment:
Why?

Reply:
Thanks I will roll back changes in expected types and verify them on Jenkins